### PR TITLE
Update Dailymotion

### PR DESCRIPTION
**Pull request by Mvlprem**
dailymotion

**Issue:**  #10073

**Similarweb rank:** [351](https://www.similarweb.com/website/dailymotion.com/#overview)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

svg taken from their careers page which is being used as a bg img look at this [screenshot](https://github.com/simple-icons/simple-icons/assets/54018894/365da812-a5a8-4b6b-bd5e-1db3d21e310a), **Hex** taken from stylesheet ` #0A0A0A `

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3486,7 +3486,7 @@
         {
             "title": "Dailymotion",
             "hex": "0D0D0D",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:3488]:
```suggestion
            "hex": "0A0A0A",
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Incredible find, @Mvlprem! 😁 

There are, though, a few excess points in the original which you might be able to remove in your version. Would you mind having a look at that?

Also, from the website's stylesheet, it looks like the colour has been changed slightly so we should update that here too.

**Comment by PeterShaggyNoble** [comment]
I'm still seeing those excess points, @Mvlprem; there are 2 clusters in the bottom left and one pair of overlapping points in the upper left.

**Comment by Mvlprem** [comment]
> I'm still seeing those excess points, @Mvlprem; there are 2 clusters in the bottom left and one pair of overlapping points in the upper left.

@PeterShaggyNoble can you provide a screenshot, i didn't quite get you. Sorry for the extra work

**Comment by PeterShaggyNoble** [comment]
@Mvlprem, see the circled points [here](https://github.com/simple-icons/simple-icons/assets/15157491/7bf2efcc-8110-4aeb-b9c7-45fb6571c7f6).

**Comment by Mvlprem** [comment]
> @Mvlprem, see the circled points [here](https://github.com/simple-icons/simple-icons/assets/15157491/7bf2efcc-8110-4aeb-b9c7-45fb6571c7f6).

@PeterShaggyNoble Fixed excess points

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**